src/backend/parser/parse_relation.c

```diff
@@ -2714,12 +2714,12 @@ warnAutoRange(ParseState *pstate, RangeVar *relation, int location)
 void
 ExecCheckRTPerms(List *rangeTable)
 {
-  if (enable_ranger && !fallBackToNativeChecks(ACL_KIND_CLASS,rangeTable,GetUserId()))
-  {
-    if(rangeTable!=NULL)
-      ExecCheckRTPermsWithRanger(rangeTable);
-    return;
-  }
+	if (enable_ranger && !fallBackToNativeChecks(ACL_KIND_CLASS,rangeTable,GetUserId()))
+	{
+		if(rangeTable!=NULL)
+			ExecCheckRTPermsWithRanger(rangeTable);
+		return;
+	}
```

Comment:
Only the two function used 2space ident, and other used Tab.
So I unify them to Tab.